libs/api/domains/driving-license-book/src/drivingLicenseBook.resolver.ts

```diff
@@ -27,30 +27,39 @@ import { DrivingSchoolType } from './models/drivingLicenseBookSchoolType.respons
 import { DrivingSchoolEmployeeGuard } from './guards/drivingSchoolEmployee.guard'
 import { DrivingInstructorOrEmployeeGuard } from './guards/drivingInstructorOrEmployee.guard'
 import { ApiScope } from '@island.is/auth/scopes'
+import { StudentIdInterceptor } from './interceptors/studentId.interceptor'
+import { Audit } from '@island.is/nest/audit'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
 @Resolver()
+@Audit({ namespace: '@island.is/api/driving-license-book' })
 export class DrivingLicenseBookResolver {
   constructor(
     private readonly drivingLicenseBookService: DrivingLicenseBookService,
   ) {}
 
   @UseGuards(DrivingInstructorOrEmployeeGuard)
+  @UseInterceptors(StudentIdInterceptor)
   @Query(() => [DrivingLicenseBookStudent])
+  @Audit()
```

Comment:
Is this needed since the resolver itself is audited?

Reply:
I think so, but I'm not 100% I was following examples like here `libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts`